### PR TITLE
Fix/issue 151 bad reload links

### PR DESCRIPTION
**Pull request by Jukaro**

### CODE REVIEW

File: srcs/django_code/front/components/about.js
Comment (ltuffery) [inline srcs/django_code/front/components/about.js:1]: Rajoute toi dedans

File: srcs/django_code/front/components/profile/quickgame.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ export class QuickGame extends Component
                 <span class="badge rounded-pill ${bgColor}">${targetUser.score} - ${opponent.score}</span>
             </div>
             <div class="col ms-2 fw-bold text-end">
-                <a href="/user/${opponent.user.username}">${opponent.user.username}</a>
+                <a is="c-link" href="/user/${this.getAttribute('opponent')}">${this.getAttribute('opponent')}</a>
```
Comment (ltuffery) [inline srcs/django_code/front/components/profile/quickgame.js:28]:
```suggestion
                <a is="c-link" href="/user/${opponent.user.username}">${opponent.user.username}</a>
```

File: srcs/django_code/front/components/about.js
Comment (Jukaro) [reply srcs/django_code/front/components/about.js:1]: Dans un autre commit non ?

Comment (ltuffery) [reply srcs/django_code/front/components/about.js:1]: ok

**Review by ltuffery — APPROVED** [review]
LGFM

**Pull request merged**